<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportGeneral</class>
 <widget class="QMainWindow" name="ReportGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informe General de Ventas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ICONOI.jpg</normaloff>ICONOI.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(4,155,180, 255), stop:1 rgba(125,216,87, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="tittleGe">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>18</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255, 0);
color: rgb(255, 255, 255);
font: 87 18pt &quot;Arial Black&quot;;
</string>
    </property>
    <property name="text">
     <string>INFORME GENERAL DE VENTAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackGe">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 185, 41);
color: rgb(255, 255, 255);
border-radius:15px;
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Atr√°s</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listProd">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>181</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(125,216,87, 255), stop:1 rgba(4,155,180, 255));</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listRef">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>131</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QListWidget" name="listVlr">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>81</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QListWidget" name="listCant">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>51</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(125,216,87, 255), stop:1 rgba(4,155,180, 255));</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listSales">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>51</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="eTotal">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Total Ventas:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listRef</tabstop>
  <tabstop>listProd</tabstop>
  <tabstop>listVlr</tabstop>
  <tabstop>listCant</tabstop>
  <tabstop>listSales</tabstop>
  <tabstop>btnBackGe</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
